tests/ext/sandbox-prehook/dd_trace_function_userland.phpt

```diff
@@ -0,0 +1,41 @@
+--TEST--
+[Prehook Regression] dd_trace_function() can trace userland functions with internal spans
+--SKIPIF--
+<?php if (PHP_VERSION_ID < 70000) die('skip: Prehook not supported on PHP 5'); ?>
+--FILE--
+<?php
+use DDTrace\SpanData;
+
+var_dump(dd_trace_function('filter_to_array', ['prehook' => function (SpanData $span) {
+    $span->name = 'filter_to_array';
+}]));
+
+function filter_to_array($fn, $input) {
+    $output = array();
+    foreach ($input as $x) {
+        if (call_user_func($fn, $x)) {
```

Comment:
Are we calling `call_user_func` array because of some issue on older PHP versions or for some other specific reason? On newer ones it can definitely be called directly: 
```suggestion
        if ($fn($x)) {
```

Reply:
This is another test that was copied as-is from the original: https://github.com/DataDog/dd-trace-php/blob/master/tests/ext/sandbox/dd_trace_function_userland.phpt#L16 But I'm 👍 for the change! :)